vllm/v1/outputs.py

```diff
@@ -114,6 +114,34 @@ class ModelRunnerOutput:
     num_nans_in_logits: Optional[dict[str, int]] = None
 
 
+# ModelRunnerOutput wrapper for async scheduling.
+# Contains GPU tensors which must be serialized before sending
+# to the scheduler process.
+@dataclass
+class AsyncModelRunnerOutput:
+    model_runner_output: ModelRunnerOutput
+
+    # [num_reqs, max_num_generated_tokens]
+    sampled_token_ids: torch.Tensor
+
+    invalid_req_indices: list[int]
+
+    def serialize(self, copy_stream: torch.cuda.Stream) -> ModelRunnerOutput:
```

Comment:
Perhaps name this something different like `get_output()`? Since it's not actually serializing